Dockerfile

```diff
@@ -20,6 +20,8 @@ FROM scratch
 COPY --from=builder /app/main /app/main
 COPY --from=builder /app/www /www
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY extras/items.csv.gz /app/
 
-# Run the hello binary.
-ENTRYPOINT ["/app/main"]
+WORKDIR /app
+# Run the binary.
+ENTRYPOINT ["/app/main", "--csv", "items.csv.gz"]
```

Comment:
what happens if items.csv.gz is not set?

Reply:
prints a log line that file is missing.

Reply:
docker files cannot be build without items.csv I will comment that stuff out.